hardware_interface/source/hyperion.c

```diff
@@ -68,66 +68,76 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
 
     switch (channel) {
     /* Channel 1 to 3 handle temp sensor */
-    case CONFIG_1_CHANNEL_TEMP_1:
+    case CONFIG_1_CHANNEL_TEMP_1:{
         adc_init(slave_addr, 1 << 7);
```

Comment:
What do these numbers mean? Should each of these 1 << <n> be their own macro? I suspect they should

Reply:
I added the macros.